packages/oats-koa-adapter/package.json

```diff
@@ -20,7 +20,7 @@
     "url": "https://github.com/smartlyio/oats.git"
   },
   "peerDependencies": {
-    "@smartlyio/oats-runtime": "^2.17.0",
+    "@smartlyio/oats-runtime": "^3.1.1",
```

Comment:
@ganymedes01, any suggestions what to do with peerdeps+lerna

Reply:
welp https://stackoverflow.com/questions/61864385/how-to-automatically-bump-version-of-peerdependencies-for-local-packages-in-a-mo